docker/models/containers.py

```diff
@@ -709,7 +709,15 @@ def run(self, image, command=None, stdout=True, stderr=False,
         if exit_status != 0:
             stdout = False
             stderr = True
-        out = container.logs(stdout=stdout, stderr=stderr)
+
+        config = self.client.api.inspect_container(container.id)
```

Comment:
Rather than make an additional call to the API, we should be able to determine the driver from `kwargs['log_config']`